Cargo.lock

```diff
@@ -1679,6 +1737,19 @@ dependencies = [
  "tokio-rustls",
 ]
 
+[[package]]
```

Comment:
When you switch to `ureq`, please be sure to only use the minimum features required to solve the problem (e.g. JSON support and that's it).  No need for cookies or TLS -- the Stacks node supports neither anyway.